lib/katello/repository_types/python.rb

```diff
@@ -34,4 +34,5 @@
                        pulp3_service_class: Katello::Pulp3::GenericContentUnit,
                        removable: true,
                        uploadable: true
+  default_managed_content_type Katello::GenericContentUnit
```

Comment:
Let me explain the history of this argument.  Originally there was no way for the user to pass in a content type to the api to remove content or upload content.  The user would just say for repository x, remove content Y.  This meant that only one type of content was supported for removal/uploading.  We added the ability to pass in the type, but needed to keep api compatibility so we added this option to specify the default if no type is passed in. 

So here, i don't think passing in Katello::GenericContentUnit makes sense, consider the possibility of a repo having 2 different content units.    We may need to modify this method to take a label instead of a class, so now we'd have:
```
default_managed_content_type :python_package
```
we'd need to update that method to handle it, and probably modify the other instances of it to use the symbol instead of the class